graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -64,6 +64,26 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
       const torch::optional<torch::Tensor>& node_type_offset,
       const torch::optional<torch::Tensor>& type_per_edge);
 
+  /**
+   * @brief Performs neighbor sampling for a given set of seed nodes taking edge
+   * type into account, where each edge type has a specified pick number.
+   *
+   * @param seed_nodes The tensor containing the seed nodes.
+   * @param fanouts The tensor containing the number of neighbors to sample per
+   * edge type.
+   * @param replace Boolean indicating if sampling is done with replacement.
```

Comment:
what's "done with replacement"? not intuitive to undersrand.

Reply:
add more explaination.